src/components/common/Filter.js

```diff
@@ -260,6 +268,13 @@ function Filter(props) {
           </Box>
         </Box>
       </form>
+      {showMessage ? (
+        <Alert severity="warning" sx={{ my: 4 }}>
+          Start date: {startDate} is bigger than end date: {endDate}
+        </Alert>
+      ) : (
+        <p></p>
+      )}
```

Comment:
another place for short-circuiting

```jsx
     {showMessage && (
        <Alert severity="warning" sx={{ my: 4 }}>
          Start date: {startDate} is bigger than end date: {endDate}
        </Alert>
      )}
```

Reply:
I did this in the newest changes, haha !